src/main/java/com/snowypeaksystems/mobactions/listener/EventListener.java

```diff
@@ -61,14 +62,21 @@ public void onMobDamage(EntityDamageEvent event) {
       return;
     }
 
-    IInteractiveMob mob = ma.getInteractiveMob((LivingEntity) event.getEntity());
-    if (event instanceof EntityDamageByEntityEvent
-        && ((EntityDamageByEntityEvent) event).getDamager() instanceof Player) {
-      MobActionsUser user = ma.getPlayer((Player) ((EntityDamageByEntityEvent) event).getDamager());
-      processEvent(user, mob, event);
-    } else if (mob.exists()) {
-      event.setCancelled(true);
+
+
+    if (event instanceof EntityDamageByEntityEvent) {
+      IInteractiveMob mob = ma.getInteractiveMob((LivingEntity) event.getEntity());
+      Entity damager = ((EntityDamageByEntityEvent) event).getDamager();
+
+      if (damager instanceof Player) {
+        MobActionsUser user = ma.getPlayer((Player) damager);
+        processEvent(user, mob, event);
+      } else if (mob.exists() || ma.getInteractiveMob((LivingEntity) damager).exists()
+          && event.getEntity() instanceof Player) {
```

Comment:
This will actually never be true because of line 61. I could be wrong but I think that will be an issue.